lib/matplotlib/axes/_axes.py

```diff
@@ -2394,7 +2394,8 @@ def stem(self, *args, **kwargs):
                 linestyle, linemarker, linecolor = \
                     _process_plot_format(linefmt)
         else:
-            linestyle, linemarker, linecolor = _process_plot_format(linefmt)
```

Comment:
I actually returned it to the way that it originally was (after I changed it). I guess the issue around these lines with the width. Is it then preferred to let lines run over 79/80 chars when it can't be helped? 

And what about multi-line strings? Should they be wrapped in ( ) instead of backslashes?